### ISSUE TITLE
Support bazel in shipshape

### ISSUE THREAD

**Issue by ciera** [google/shipshape#73]
Right now, we're only supporting maven builds. Kythe now also supports using bazel.

**Comment by emsod** [comment]
The extractor image we want to use lives here: https://github.com/google/kythe/tree/master/kythe/java/com/google/devtools/kythe/extractors/java/bazel

**Comment by clconway** [comment]
This is specifically about support Bazel _builds_ for Shipshape _analyses_, right? And not for building Shipshape with Bazel?

**Comment by clconway** [comment]
By "support [for] Bazel builds for Shipshape analyses", I mean analyzing code built with Bazel, if that's not clear.

**Comment by ciera** [comment]
@clconway Correct, this is for making it possible to analyze codebases built with bazel.
